crates/epaint/src/tessellator.rs

```diff
@@ -1440,6 +1440,14 @@ impl Tessellator {
 
         // If the arc is a full circle, we can just use the circle function.
         if (end_angle - start_angle).abs() >= std::f32::consts::TAU {
+            let stroke_color = match stroke.color {
+                ColorMode::Solid(color) => color,
+                ColorMode::UV(callback) => {
+                    // We can't use the callback here, so we just use the center of the circle.
```

Comment:
Maybe this behavior should be documented somewhere? I'm also not fully sure if you can't use the callback here, could you elaborate?

Reply:
The entire pie is rendered as a circle, but `CircleShape` is not compatible with `PathStroke`. 
As a workaround, I use a callback to obtain the center color from `PathStroke`. 
This issue should be addressed once `CircleShape` supports `PathStroke`.

Reply:
Ah, I understand. Thanks for elaborating!

Im hoping that the underlying `ColorMode` will be used for more shapes in the future, but for the time being it is only for paths. If you want, I can look into opening a PR to support `PathStroke` for more shapes